Snowflake.Data.Tests/IntegrationTests/SFDbDataReaderIT.cs

```diff
@@ -811,6 +815,8 @@ public void TestGetChar()
                 // Act
                 using (IDataReader reader = cmd.ExecuteReader())
                 {
+                    ValidateResultFormat(reader);
```

Comment:
this validation should be part of assert block

Reply:
What do you think about Assert.IsTrue(reader.Read()) ?
I think it should be in Act not Assert block. But if so, should we leave Assert.IsTrue(...) or just call reader.Read() ?

Reply:
ValidateResultFormat is a wraper for:
```
Assert.AreEqual(_resultFormat, ((SnowflakeDbDataReader)reader).ResultFormat);
```

which is definitely an assert.

If you want you could split `Assert.IsTrue(reader.Read())` into something like :
```
// act
var read = reader.Read();

// assert
Assert.IsTrue(read);
```

you can split that. However I would not do it, because for me "act" block should be "execute query on the server and get the result" and that's `cmd.ExecuteReader()` and checking the result is rather a part of assert for me.